src/class-elements/grammar-privatename-no-receiver-error-accessor-set-meth.case

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: It is syntax error if IdentifierName is accessed without receiver (Accessor set Method)
+info: |
+  Updated Productions
+
+  ClassElementName :
+    PropertyName
+    PrivateName
+
+  PrivateName ::
+    # IdentifierName
+template: syntax/invalid
+features: [class-methods-private]
+---*/
+
+//- elements
+set #m(_) {}
+method() {
+#m();
```

Comment:
like the other case, `#m` or maybe cloning this case file to a second one with `#m = 1` should also be nice.

Reply:
none would be evaluated in runtime but should not hurt adding both cases.